source/octf/trace/parser/ParsedIoTraceEventHandler.h

```diff
@@ -99,6 +113,7 @@ class ParsedIoTraceEventHandler
     uint64_t m_subrangeStart;
     uint64_t m_subrangeEnd;
     std::map<uint64_t, IoQueueDepth> m_devIoQueueDepth;
+    std::map<uint64_t, FileSystemViewer> m_fsViewers;
```

Comment:
I suggest naming these "m_partitionViewers" or something like that to give information about keys meaning.